applications/nrf_desktop/Kconfig.ble

```diff
@@ -28,19 +28,26 @@ config DESKTOP_BT_PERIPHERAL
 	bool
 	default y if DESKTOP_ROLE_HID_PERIPHERAL
 	imply DESKTOP_BLE_ADV
+	imply DESKTOP_BLE_LATENCY_ENABLE
 	imply DESKTOP_BAS_ENABLE
 	imply DESKTOP_HIDS_ENABLE
 	imply BT_DIS
+	imply DESKTOP_DEV_DESCR_ENABLE
```

Comment:
It would be great to indicate that this module belongs to the Bluetooth category as it does not seem so at the first glance. Consider creating a task for it as I don't want to extend the scope of this task:

```DESKTOP_DEV_DESCR_ENABLE```
->
```DESKTOP_BLE_DDS_ENABLE``` (Device description service)

Reply:
Added follow-up task: `NCSDK-19651`